plugins/item_previews/web_client/views/ItemPreviewWidget.js

```diff
@@ -66,27 +71,29 @@ var ItemPreviewWidget = View.extend({
             isJSONItem: this._isJSONItem
         }));
 
-        // Render any JSON files.
-        supportedItems.filter(this._isJSONItem).forEach(function (item) {
-            var id = item.get('_id');
-
-            // Don't process JSON files that are too big to preview.
-            var size = item.get('size');
-            if (size > this._MAX_JSON_SIZE) {
-                return view.$('.json[data-id="' + id + '"]').text('JSON too big to preview.');
-            }
-
-            // Ajax request the JSON files to display them.
-            restRequest({
-                path: 'item/' + id + '/download',
-                type: 'GET',
-                error: null // don't do default error behavior (validation may fail)
-            }).done(function (resp) {
-                view.$('.json[data-id="' + id + '"]').text(JSON.stringify(resp, null, '\t'));
-            }).error(function (err) {
-                console.error('Could not preview item', err);
+        if (!this.wrapPreviews) {
```

Comment:
Removing this leads to an error since `supportedItems` is not a function and we call `supportedItems.filter' ligne 76